component/src/lib.rs

```diff
@@ -48,6 +48,15 @@ impl Component {
             .ok_or_else(|| anyhow!("component with name '{}' does not exist", name))
             .and_then(|c| Ok(c.clone()))
     }
+
+    pub fn is_default_forc_plugin(name: &str) -> bool {
+        (Self::from_name(FORC)
+            .unwrap()
+            .executables
+            .contains(&name.to_string())
+            && name != FORC)
+            || name == "forc-client"
```

Comment:
same here

Reply:
Hmm, I didn't want to create a const for just 1 check at a place - the line of thought was that this could cause future PRs/contributions to use this `FORC_CLIENT` const should we add it to `constants.rs`, when this is just a one time check, plus it's probably unlikely that the name `forc-client` would be changed frequently anyway :thinking:

Reply:
Using `const`s for str literals like this is not only about ease of refactoring or renaming, but also about removing the scope for human error when inevitably we want to use the same string again somewhere else.

E.g. it can be easy to miss a typo in a string literal, but you cannot write a typo in a `const` binding without getting a compile error. By maintaining a policy of using `const`s for literals rather than repeating the literals themselves, you only have to check for typos in one location.

> I didn't want to create a const for just 1 check at a place

We use the same literal above in the `executables` iteration :smiling_face_with_tear:

Reply:
You make a good point, will change it accordingly :+1: